engines/script/sandbox.go

```diff
@@ -58,7 +81,11 @@ func (s *sandbox) run() {
 			s.monitor.Errorf("Failed to remove temporary folder, error: %s", err)
 		}
 
-		s.resultSet = &resultSet{success: success}
+		if resultError == nil {
+			s.resultSet = &resultSet{success: success}
+		} else {
+			s.resultError = resultError
+		}
```

Comment:
I am confused here, shouldn't `result` set be included always set and in case `resultSet.sucess == false` `resultError` be set too?

Reply:
Nope, not if there is an internal error... then there is no `ResultSet`